.github/workflows/rancher-components.yaml

```diff
@@ -0,0 +1,25 @@
+name: rancher-components
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  validate:
```

Comment:
Please split this job in `build` and `lint` as we want to have separated checks and know what is failing in a glance